packages/react/src/hooks/contracts/usePrepareContractWrite.test.ts

```diff
@@ -324,6 +324,7 @@ describe('usePrepareContractWrite', () => {
       config[property as keyof typeof config] =
         baseConfig[property as keyof typeof baseConfig]
       rerender()
+      await actConnect({ utils })
```

Comment:
Interesting. Any idea why we need to call `actConnect` again now? Shouldn't the Client already be connected?

Reply:
Yeah, it's a weird one. The client is already connected, and I even tested this E2E, but not sure why it isn't working here. Might have another look.